### ISSUE TITLE
Add a nodejs REPL command and/or example

### ISSUE THREAD

**Issue by emezeske** [emezeske/lein-cljsbuild#81]
@swanodette already has an almost-working setup:

https://github.com/swannodette/clojurescript/blob/node-repl/script/node-repl

**Comment by dahjelle** [comment]
FWIW, it looks like @bodil is no longer maintaining that plugin, [as per this comment](https://github.com/bodil/cljs-noderepl/issues/21#issuecomment-57473682).

**Comment by bodil** [comment]
AFAIK it's also been superseded by functionality in Cljs core?

**Comment by dahjelle** [comment]
@bodil, such as in [this bit of the Quick Start guide](https://github.com/clojure/clojurescript/wiki/Quick-Start#nodejs-repl)? Yes, and that works…just doesn't have the nice PATH handling of cljsbuild, does it?

I'm rather a n00b here so any tooling suggestions very welcome. :-)

**Comment by dahjelle** [comment]
I see this is addressed in pull request #379.

**Comment by dahjelle** [comment]
In case anyone else ends up here from a search, I found this to work reasonably well:

1. Create a `repl.clj` with the following contents (as on the [Quick Start](https://github.com/clojure/clojurescript/wiki/Quick-Start#running-clojurescript-on-nodejs)):

```clojure
(require 'cljs.repl)
(require 'cljs.build.api)
(require 'cljs.repl.node)

(cljs.build.api/build "src"
  {:main 'hello-world.core
   :output-to "out/main.js"
   :verbose true})

(cljs.repl/repl (cljs.repl.node/repl-env)
  :watch "src"
  :output-dir "out")
```

Add the following alias to your `project.clj`:

```clojure
  :aliases {"noderepl" ["run" "-m" "clojure.main" "repl.clj"]}
```

Now, you can get a Node.js REPL via

```sh
$ rlwrap lein noderepl
```
